pyproject.toml

```diff
@@ -139,7 +139,7 @@ docs = [
 ]
 dynamodb = [
   "boto3",
-  "commons-codec>=0.0.12",
+  "commons-codec @ git+https://github.com/crate/commons-codec.git@dynamodb-full-load-batch",
```

Comment:
`commons-codec` needs to converge into a release beforehand.

Reply:
[commons-codec v0.0.14](https://github.com/crate/commons-codec/releases/tag/v0.0.14) has been released.